examples/material_gallery/lib/gallery/home.dart

```diff
@@ -50,13 +50,15 @@ class GalleryHome extends StatefulWidget {
 }
 
 class GalleryHomeState extends State<GalleryHome> {
-  final Key _scrollableKey = new UniqueKey();
+  final Key _homeKey = new ValueKey("Gallery Home");
+  final Key _listKey = new ValueKey("Gallery List");
```

Comment:
ValueKey is a bit dubious here. Shouldn't this just be a couple of static `UniqueKey`s?